library/sanitize.inc.php

```diff
@@ -189,3 +189,21 @@ function sanitizeNumber($number)
         return $clean_number;
     }
 }
+// Get contents over https
+function file_get_contents_curl($url) 
+{
+    $ch = curl_init();
+
+    curl_setopt($ch, CURLOPT_AUTOREFERER, true);
+    curl_setopt($ch, CURLOPT_HEADER, 0);
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+    curl_setopt($ch, CURLOPT_URL, $url);
+    curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true); 
+    curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, 2); 
+    curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);     
+
+    $data = curl_exec($ch);
+    curl_close($ch);
+
+    return $data;
+}
```

Comment:
Rec keeping the function in this library since other security related stuff is here and it is always loaded in the openemr scripts.

Reply:
Special thanks to @danehrlich1 for this